5.sc_Morphology_Logistic_Regression/1.train_models/0.train_single_class.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    grid_search_cv = GridSearchCV(
</pre><p>You could also use this:</p><p>https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegressionCV.html#sklearn.linear_model.LogisticRegressionCV</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I was looking into this and I think I might use it in the future, but Roshan already had code using the <code>GridSearchCV</code> so I ran with it. </p><p> </p>